### ISSUE TITLE
Doesnt remove generic type

### ISSUE THREAD

**Issue by wolverineks** [flowtype/flow-remove-types#77]
I was expecting that using `yarn flow-remove-types deleteme.js --all --pretty -d no-types`

given:

```js
/* @flow */

function foo<T> (a: T): T {
  return a
}

const bar = foo<number>(123)
```
the output would be:

```js
/*  */

function foo (a) {
  return a
}

const bar = foo(123)
```

but instead i got:

```js
/*  */

function foo (a) {
  return a
}

const bar = foo<number>(123)
```

Is this the expected behavior? 
Is it possible to remove the generic types with this utility?

**Comment by motiz88** [comment]
This will be resolved in the next major release which will upgrade the parser to recognise this syntax.

**Comment by motiz88** [comment]
Closed in #82.

**Comment by motiz88** [comment]
Closing in favour of #89.
